traitlets/traitlets.py

```diff
@@ -547,6 +547,51 @@ def tag(self, **metadata):
 # The HasTraits implementation
 #-----------------------------------------------------------------------------
 
+class _CbWrapper(object):
```

Comment:
Can we make this `_CallbackWrapper`?  It feels more pythonic than using abbreviations.